internal_deps.bzl

```diff
@@ -4,37 +4,61 @@ Users should *not* need to install these. If users see a load()
 statement from these, that's a bug in our distribution.
 """
 
-# buildifier: disable=bzl-visibility
-load("//cypress/private:maybe.bzl", http_archive = "maybe_http_archive")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", _http_archive = "http_archive")
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+
+def http_archive(name, **kwargs):
+    maybe(_http_archive, name = name, **kwargs)
 
 def rules_cypress_internal_deps():
     "Fetch deps needed for local development"
+    http_archive(
+        name = "build_bazel_integration_testing",
```

Comment:
the `build_bazel_integration_testing` dep can go; we don't use it anywhere in our rules. Should probably remove from the upstream rules_template

Reply:
Done